swan-cern/scripts/11_htcondor.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino 2023
+# Copyright CERN
+# The HT Condor environment is condfigured here.
+
+# HTCondor at CERN integration
```

Comment:
More comments as spark.

Reply:
Added a comment further explaining what the script does.